Source/DafnyStandardLibraries/src/dfyconfig.toml

```
@@ -1,4 +1,7 @@
 
+# *Nonlinear.dfy in Arithmetic/Internals/ are subject to the options here
+excludes = ["**/Arithmetic/*.dfy", "**/Arithmetic/Internals/*Internals.dfy"]
```

```suggestion
excludes = ["**/Arithmetic/*.dfy", "**/Arithmetic/**/*Internals.dfy"]
```

Just to keep the two project files 100% in sync. Not THAT critical because if we double-include or miss files, we'll get errors when testing the examples, but better to avoid the possibility. :)